### DISCUSSION TITLE
att&ck extractor

### DISCUSSION THREAD

**Discussion by freyntjens** [d3fend/d3fend-ontology#182] [General]
Hi all,

Is there any way to visualize the result of att&ck extractor on the navigator as we can on the d3fend extractor?

Thanks in advance,

Fred

**Comment by netfl0** [comment]
Great idea, we can get this on the list.

**Reply by freyntjens** [reply]
Hi Peter,

Thanks for the swift reply!
It would be really great, I'm currently working on a methodology to address new attack techniques for witch reports are already produced,  hence being able to visualize TTP's used on the matrix  would be really helpful to prioritize defense tactics.
Thanks,

Fred
